### ISSUE TITLE
Query fails with NaN if string provided

### ISSUE THREAD

**Issue by connor4312** [balderdashy/sails-mysql#182]
If you provide a string in a "where" clause that expects an integer, it'll insert NaN into the query and cause a 500 error to be thrown.

This is obviously not ideal at all. It's a major pain having to check that every single input is numeric before passing it to the WHERE - it should be handled by just returning no results.

Either:
 * The query value should always be parameterized. This is probably the preferred fix, MySQL will handle the input correctly (just not return any matching results, because there are none!) or
 * Manually return an empty array of results.

**Comment by devinivy** [comment]
Does this bug persist?

**Comment by sailsbot** [comment]
Thanks for posting, @connor4312.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
